src/main/java/com/hpe/adm/octane/ideplugins/services/EntityService.java

```diff
@@ -129,7 +137,20 @@
             }
         }
 
-        getRequest.addOrderBy("id", true);
+        if (offset != null) {
+            getRequest = getRequest.offset(offset);
+        }
+
+        if (limit != null) {
+            getRequest = getRequest.limit(limit);
```

Comment:
@atothhpe Off-topic: Why does the limit() method need to return an instance of the owning object?

Reply:
SDK design decision, it's supposed to be the builder pattern. So you can chain the functions together. Overwriting the reference is optional here, since the limit function will modify the request internally.